ExpectedColorMap.h

```diff
@@ -0,0 +1,33 @@
+#include <string>
+#include <iostream>
+#ifndef EXPECTEDCOLORMAP_H
+#define EXPECTEDCOLORMAP_H
+
+std::string expectedColorMap = 

```

Comment:
In the expected color map, the column separators are still not aligned. When a test "passes", it needs to give you confidence that the user finds your solution "acceptable". Try it out